pydantic_ai_slim/pydantic_ai/mcp.py

```diff
@@ -866,6 +867,75 @@ def __eq__(self, value: object, /) -> bool:
         return self.url == value.url
 
 
+class MCPServerWebSocket(_MCPServerHTTP):
```

Comment:
Should this inherit from `MCPServer` instead of `_MCPServerHTTP` if it overrides most of what that class does?